evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -1639,8 +1641,13 @@ def __init__(self, parameters, random_seed=0):
     )
 
     # Test fitted equality
-    X = pd.DataFrame({})
-    y = pd.Series([])
+    if pipeline_class == BinaryClassificationPipeline:
+        X, y = X_y_binary
+    elif pipeline_class == MulticlassClassificationPipeline:
+        X, y = X_y_multi
+    else:
+        X, y = X_y_regression
```

Comment:
With the number of times you've had to add this logic to tests, it might be worth it to add a util function somewhere to grab the test data based on the problem type.

Reply:
yes